tests/datavolume_test.go

```diff
@@ -1246,30 +1246,12 @@ var _ = Describe("[vendor:cnv-qe@redhat.com][level:component]DataVolume tests",
 				}}),
 		)
 
-		DescribeTable("should have an alert suppressing label on corresponding PVC", func(dvFunc func(string, string, string) *cdiv1.DataVolume, url string) {
-			dataVolume := dvFunc("suppress-fillingup-alert-dv", "1Gi", url)
-
-			By(fmt.Sprintf("creating new datavolume %s", dataVolume.Name))
-			dataVolume, err := utils.CreateDataVolumeFromDefinition(f.CdiClient, f.Namespace.Name, dataVolume)
-			Expect(err).ToNot(HaveOccurred())
-
-			// verify PVC was created
-			By("verifying pvc was created and is Bound")
-			pvc, err := utils.WaitForPVC(f.K8sClient, dataVolume.Namespace, dataVolume.Name)
-			Expect(err).ToNot(HaveOccurred())
-
-			// Alert-suppressing label exists
-			Expect(pvc.Labels[common.KubePersistentVolumeFillingUpSuppressLabelKey]).To(Equal(common.KubePersistentVolumeFillingUpSuppressLabelValue))
-		},
-			Entry("[test_id:8043]for import DataVolume", utils.NewDataVolumeWithHTTPImport, tinyCoreIsoURL()),
-			Entry("[test_id:8044]for upload DataVolume", createUploadDataVolume, tinyCoreIsoURL()),
-			Entry("[test_id:8045]for clone DataVolume", createCloneDataVolume, fillCommand),
-		)
-
-		DescribeTable("Should pass all DV labels and annotations to the PVC", func(dvFunc func(string, string, string) *cdiv1.DataVolume, url string) {
-			dataVolume := dvFunc("pass-all-labels-dv", "1Gi", url)
+		DescribeTable("should have an alert suppressing label and inherit labels & annotations from DV on corresponding PVC", func(dvFunc func(string, string, string) *cdiv1.DataVolume, url string) {
+			dataVolume := dvFunc("dv-labels-behaviour", "1Gi", url)
 			dataVolume.Labels = map[string]string{"test-label-1": "test-label-1", "test-label-2": "test-label-2"}
 			dataVolume.Annotations = map[string]string{"test-annotation-1": "test-annotation-1", "test-annotation-2": "test-annotation-2"}
+			// Stir things up with this non widely used access mode
```

Comment:
Do you plan to add more testing?

Reply:
Nope, as you can see I am even not adding new tests, rather, integrating RWOP into existing ones
I think we shouldn't be too worried about people using RWOP, just allowing it and testing a few select flows